Maddori.Apple/Maddori.Apple/Global/UIComponent/MemberCollectionView.swift

```diff
@@ -68,10 +69,12 @@ final class MemberCollectionView: UIView {
 
 extension MemberCollectionView: UICollectionViewDelegate {
     func collectionView(_ collectionView: UICollectionView, didSelectItemAt indexPath: IndexPath) {
-        
-        // FIXME: - 추후 print문 삭제하고 navigate하는 코드 넣을 예정
-        
-        print(memberList[indexPath.item])
+        if selectedMemberList.contains(memberList[indexPath.item]) {
+            return
+        } else {
+            selectedMemberList.append(memberList[indexPath.item])
+        }
```

Comment:
p4;
```suggestion
        if !selectedMemberList.contains(memberList[indexPath.item]) {
            selectedMemberList.append(memberList[indexPath.item])
        }
```
같은 맥락이라면 이렇게 바꿀 수 있을 것 같습니다 !

Reply:
[356be0f](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/61/commits/356be0f84512916f2d41a3711d3c6d984b34f108)
오 이렇게 적으면 되는군요 ㅋㅋㅋ 이 부분 적으면서 저도 좀 이상하다 싶었거든요,,,,
수정해서 반영해두었습니다 ! 감사합니다 :)